daemon/info.go

```diff
@@ -174,7 +175,13 @@ func (daemon *Daemon) showPluginsInfo() types.PluginsInfo {
 
 	pluginsInfo.Volume = volumedrivers.GetDriverList()
 	pluginsInfo.Network = daemon.GetNetworkDriverList()
-	pluginsInfo.Authorization = daemon.configStore.AuthorizationPlugins
+
+	authPlugins := []string{}
+	for _, p := range daemon.configStore.AuthorizationPlugins {
+		authPlugins = append(authPlugins, p)
+	}
+	sort.Strings(authPlugins)
+	pluginsInfo.Authorization = authPlugins
```

Comment:
Yes, Now I updated the PR like that. Since `AuthorizationPlugins ` is a common config among all platforms, so I located the function `GetAuthorizationPlugins ` in config.go, PTAL, thanks.